src/lib/server/session.ts

```diff
@@ -1,13 +1,8 @@
-
 import { verifyToken } from '@clerk/backend'
 
 export const verifySession = async (secretKey: string, sessionToken: string) => {
 	if (sessionToken) {
-		const issuer = (issuer: string) => issuer.startsWith('https://clerk.') || issuer.includes('.clerk.accounts')
-		const claims = await verifyToken(sessionToken, {
-			secretKey,
-			issuer,
```

Comment:
TS error coz `issuer` is removed for some reason

Reply:
Updating to use https://clerk.com/docs/references/backend/authenticate-request#authenticate-request